src/test/java/de/htwberlin/madlib/MadLibTest.java

```diff
@@ -0,0 +1,31 @@
+package de.htwberlin.madlib;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.Arrays;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class MadLibTest {
+    MadLib testObject = new MadLib();
+    @Test
+    @DisplayName("The result string should starts with \"Java programming is so\" ")
+    void startWithPattern() {
+        String expectedStartWith = "Java programming is so";
+        boolean actual = testObject.create().startsWith(expectedStartWith);
+        assertEquals(true,actual);
```

Comment:
Ginge etwas einfacher mit:

```suggestion
        assertTrue(actual);
```